Cubical/Functions/Image.agda

```diff
@@ -0,0 +1,75 @@
+{-# OPTIONS --safe #-}
+module Cubical.Functions.Image where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Structure
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.Equiv.Fiberwise
+open import Cubical.Foundations.Univalence
```

Comment:
A lot of imports for a small file. Do you really need all of them?

Reply:
Your suspicion was correct - not all of them were needed (anymore..).